### PR TITLE
Revise descriptions for tasks

### PR DESCRIPTION
**Pull request by sftim**
Add description metadata for each task section (English localization).
For sections that only have one page, make sure that that sole page has description metadata set.

[Preview](https://deploy-preview-21752--kubernetes-io-master-staging.netlify.app/docs/tasks/)

/kind cleanup

### CODE REVIEW

File: content/en/docs/tasks/extend-kubectl/kubectl-plugins.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Extend kubectl with plugins
 reviewers:
 - juanvallejo
 - soltysh
-description: With kubectl plugins, you can extend the functionality of the kubectl command by adding new subcommands.
+description: Extend the functionality of the kubectl command by adding new subcommands.
```
Comment (kbhawkey) [inline content/en/docs/tasks/extend-kubectl/kubectl-plugins.md:6]: nit: The plugin part is the main concept. Something like:
Install, write, and distribute kubectl extension plugins.
or
Extend kubectl by creating and installing kubectl plugins.

Comment (sftim) [reply content/en/docs/tasks/extend-kubectl/kubectl-plugins.md:6]: Think this is good to fix after we merge?

Comment (sftim) [reply content/en/docs/tasks/extend-kubectl/kubectl-plugins.md]: I've taken the second suggestion.

**Comment by sftim** [comment]
/hold
/retitle Revise descriptions for tasks

Holding because I'd like to see this preview OK via Netlify before a merge

**Comment by sftim** [comment]
https://deploy-preview-21752--kubernetes-io-master-staging.netlify.app/docs/tasks/ is there now.
/hold cancel

**Comment by zacharysarah** [comment]
@sftim Very nice! ✨ 

/lgtm
/approve

**Pull request merged**